pt_BR.json

```diff
@@ -0,0 +1,772 @@
+  "JOIN_SESSION": "Entrar na Sessão",
+  "JOIN_SESSION_INFO": "Entrar em Informações da Sessão",
+  "KARMA": "Carma",
+  "KICK": "Chute",
+  "KICKED": "Chutado",
+  "KICK_ALL_FROM_INTERIORS": "Chutar todos de interiores",
+  "KICK_BY_RID": "Kick by RID",
+  "KICK_BY_USERNAME": "Chutar por nome de usuário",
+  "KICK_HOST_ON_JOIN": "Expulsar o host durante a entrada",
+  "KILL_ALL": "Matar todos",
+  "KILL_LIST": "lista de mortes",
+  "KILL_LOOP": "Kill Loop",
+  "KING_OF_THE_CASTLE": "rei do castelo",
+  "LANGUAGE": "Linguagem",
+  "LANGUAGE_FILTER_DESC": "Definir um filtro de região correto para o idioma ajudará tremendamente",
+  "LAW_ABIDING": "Cumpridor da lei",
+  "LEFT": "Esquerda",
+  "LEFT_SIDED": "Lado Esquerdo",
+  "LESSEN_BREAKUP_KICK": "Reduza os pontapés de separação como anfitrião",
+  "LESSEN_BREAKUP_KICK_DESCRIPTION": "O invasor deve ingressar depois que você se tornar host para que isso funcione. ",
+  "LOAD": "Carregar",
+  "LOAD_VEHICLE": "Veículo de Carga",
+  "LOCAL_TIME": "Horário local",
+  "LOCAL_WEATHER": "Tempo localizado",
+  "LOG": "Registro",
+  "LOG_CHAT_MSG": "Registrar mensagens de bate-papo",
+  "LOG_TXT_MSG": "Registrar mensagens de texto",
+  "LOW_GRIP_TIRES": "Pneus de baixa aderência",
+  "LSC_CHROME": "cromada ",
+  "LSC_STYLE": "Estilo ",
+  "LTS": "LTS",
+  "MATCHMAKING": "Matchmaking",
+  "MATCHMAKING_FAIL": "Matchmaking falhou",
+  "MATTE": "fosco",
+  "MAX": "Máximo",
+  "MAX_VEHICLE": "Veículo máximo",
+  "MC_TELEPORT": "MC Teletransporte",
+  "MELEE": "corpo a corpo",
+  "MERRYWEATHER": "Clima alegre",
+  "METALS": "Metais",
+  "MIN": "Mínimo",
+  "MINUTE": "Minuto",
+  "MISC": "Diversos",
+  "MOBILE": "Móvel",
+  "MOD": "mod",
+  "MODEL_NAME": "Nome do modelo",
+  "MODIFY_TIME": "Modificar hora",
+  "MODIFY_TIME_HOST_REQUIRED": "A modificação do tempo requer o host da sessão",
+  "MODIFY_TIME_NO_PLAYER_TIMESTAMP": "Ainda não temos o timestamp do jogador",
+  "MOD_OPTIONS": "Opções de modificação",
+  "MORS_FIX_ALL": "Mors Mutual Fix Todos os Veículos",
+  "MORS_MUTUAL": "Mors Mutual",
+  "MSG_SENT_FAIL": "Mensagem não enviada. ",
+  "MSG_SENT_SUCCESS": "mensagem enviada com sucesso",
+  "MW_AIRSTRIKE": "Solicitar ataque aéreo",
+  "MW_AMMO_DROP": "Solicitar queda de munição",
+  "MW_BACKUP_HELI": "Solicitar Helicóptero Reserva",
+  "MW_HELI_PICKUP": "Recolha de Helicóptero",
+  "NAME": "Nome",
+  "NAME_VEHICLE_MODEL": "Nome do modelo do veículo",
+  "NEON": "Néon",
+  "NEON_LIGHT_OPTIONS": "Opções de luz neon",
+  "NETWORK": "Rede",
+  "NETWORK_PLAYER_MGR": "Gestor de jogadores de rede",
+  "NETWORK_PLAYER_MGR_DESTROY": "Saindo da sessão e limpando os dados do jogador.",
+  "NETWORK_PLAYER_MGR_INIT": "Entrando na sessão e inicializando os dados do jogador.",
+  "NEVER_WANTED": "Nunca Quis",
+  "NEW_ENTRY": "Nova entrada",
+  "NO": "Não",
+  "NORMAL": "Normal",
+  "NOTIFICATION_WELCOME_TEXT": "Carregado YimMenu. ",
```

Comment:
Missing format argument in `NOTIFICATION_WELCOME_TEXT` for which key to press to open YimMenu.
(Original English `Loaded YimMenu. Press {} to open`)